.github/workflows/build-conda-linux.yml

```diff
@@ -15,12 +15,13 @@ on:
 
 jobs:
   generate-matrix:
-    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@main
+    uses: pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@release/2.1
     with:
       package-type: conda
       os: linux
       test-infra-repository: pytorch/test-infra
-      test-infra-ref: main
+      test-infra-ref: release/2.1
+      channel: test
```

Comment:
This is set automatically, so no need to explicitly define.

Reply:
You mean the `ref` branch or the `channel`?  I'm trying to see which one is set automatically on https://github.com/pytorch/test-infra/blob/main/.github/workflows/generate_binary_build_matrix.yml

Reply:
normally channel: test, but in our current use case it does not matter, test config is the same as nightly